pysat/instruments/methods/general.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+"""Provides generalized routines for integrating instruments into pysat.
+"""
+
+from __future__ import absolute_import, division, print_function
+
+import datetime as dt
+import logging
+import pandas as pds
+
+
+logger = logging.getLogger(__name__)
+
+
+def convert_timestamp_to_datetime(inst):
+    """Use datetime instead of timestamp for Epoch
+
+    Parameters
+    ----------
+    inst : pysat.Instrument
+        associated pysat.Instrument object
+
+    Returns
+    -------
+    None
+        Modifies Instrument object in place
```

Comment:
Why don't we need these anymore?

Reply:
Was refreshing my knowledge of the numpy style guide recently, and it was recommended to only include a Returns block when something is returned.  The plus for this is that it makes the notes and examples in the docstring easier to read, since the entire string is shorter.